process-manager.js

```diff
@@ -139,12 +152,10 @@ class ProcessManager {
 		// To be overriden by specific implementations.
 		return '';
 	}
+
+	static get cache() {
+		return PROCESS_MANAGERS;
+	}
 }
 
-ProcessManager.cache = processManagers;
```

Comment:
Nitpicking. Defining `ProcessManager.cache` like this makes the `writable`, `enumerable`, and `configurable` values in its property descriptor all true. Changing it to a static getter in `ProcessManager` makes them all false instead, preventing things like accidentally overwriting it or deleting it harder to do. It also doesn't create a new hidden class for it after defining `ProcessManager.cache`. I can change it back if you want